kubernetes/services/api-gateway.yaml

```diff
@@ -43,7 +43,7 @@ data:
     server.max-initial-line-length=65536
 
     #refer to https://github.com/opentracing-contrib/java-spring-jaeger/tree/release-3.2.1
-    opentracing.jaeger.udp-sender.host=localhost
+    opentracing.jaeger.udp-sender.host=10.111.132.59
```

Comment:
@yanmo96 Let's remain the `localhost` in the master, and leave the modification during the deployment phase. In that way, we can modify the IP address according to the cluster's location (medina or kolla, or others).

Reply:
+1

Reply:
Thanks will put them back